src/test/script_tests.cpp

```diff
@@ -787,9 +787,10 @@ BOOST_AUTO_TEST_CASE(script_PushData)
 }
 
 CScript
-sign_multisig(CScript scriptPubKey, std::vector<CKey> keys, CTransaction transaction)
+sign_multisig(const CScript& scriptPubKey, std::vector<CKey> keys, const CTransaction& transaction, CAmount amt)
```

Comment:
I would have add an explanation of what we are trying to fix here. Form what I can understand after hard fork, replay protected tx's require the amount field to be non-zero for previous outputs in signed tx's.